models/GroupsService.php

```diff
@@ -183,16 +198,6 @@ public function cloneInstance(
 
     private function getTestTakerRootClass(): core_kernel_classes_Class
     {
```

Comment:
up to you, do you think this method is necessary anymore? :) its a one liner

Reply:
Missed it! Removed in 91646edcb8cbd209346b3388998b3c5c33f849d4, thanks!